nexus/networking/src/sled_client.rs

```diff
@@ -0,0 +1,53 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Functionality for constructing sled-agent clients.
+
+use nexus_db_queries::context::OpContext;
+use nexus_db_queries::db::lookup;
+use nexus_db_queries::db::lookup::LookupPath;
+use nexus_db_queries::db::DataStore;
+use omicron_common::api::external::Error;
+use omicron_common::api::external::LookupResult;
+use sled_agent_client::Client as SledAgentClient;
+use slog::o;
+use slog::Logger;
+use std::net::SocketAddrV6;
+use uuid::Uuid;
+
+pub fn sled_lookup<'a>(
```

Comment:
Why not use `LookupPath` directly?

Reply:
I was just moving stuff what existed without trying to make any changes. I can drop this one and keep `Nexus::sled_client` around? (That one is called a half dozen times or so; I guess we could also replace all of those with `LookupPath` invocations, but that's entirely unrelated to this PR, I think.)